rust/threshold-schnorr/src/schnorr_example_rust/tests/tests.rs

```diff
@@ -37,35 +62,74 @@ fn test_impl(pic: &PocketIc, algorithm: SchnorrAlgorithm) {
     // Make sure the canister is properly initialized
     fast_forward(&pic, 5);
 
+    let _dummy_reply: () = update(
+        &pic,
+        my_principal,
+        example_canister_id,
+        "for_test_only_change_management_canister_id",
+        encode_one(schnorr_mock_canister_id.to_text()).unwrap(),
+    )
+    .expect("failed to update management canister id");
+    // Make sure the example canister uses mock schnorr canister instead of
+    // the management canister
+    fast_forward(&pic, 5);
+
     // a message we can reverse to break the signature
     // currently pocket IC only supports 32B messages for BIP340
```

Comment:
Good point. That doesn't seem to be the case, so I removed the comment. But 32B still seems a good choice for the message length. And since the tests already take a few minutes, we probably shouldn't add more different message lengths, especially because we are already testing this API elsewhere.